src/cpp/b2w.cpp

```diff
@@ -316,15 +324,15 @@ int main(int argc, char* argv[])
         delete [] rd;
     };
 
-    if (argc == optind + 2) {          // region not specified
+    if (NULL == regionname) {          // region not specified
```

Comment:
you could also use `nullptr` here, but your call

Reply:
(given that the code is a flamboyant mix of C99 and C++17 styles :-D it's hard to deicde, I'll flip a coin)